src/plugins/ResizeReorder/ReorderHandle.js

```diff
@@ -0,0 +1,241 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReorderHandle
+ * @typechecks
+ */
+
+import React from 'react';
+import cx from 'cx';
+import PropTypes from 'prop-types';
+import DOMMouseMoveTracker from 'DOMMouseMoveTracker';
+import { getState } from 'FixedDataTableStore';
+
+const DRAG_SCROLL_SPEED = 10;
+const DRAG_SCROLL_BUFFER = 100;
+
+class ReorderHandle extends React.Component {
+
+  static propTypes = {
+    height: PropTypes.number,
+    touchEnabled: PropTypes.bool,
+    isRTL: PropTypes.bool,
+    scrollX: PropTypes.number,
+    maxScrollX: PropTypes.number,
+    left: PropTypes.number,
+    availableScrollWidth: PropTypes.number,
+    isFixed: PropTypes.bool,
+    scrollToX: PropTypes.func,
+    onColumnReorderEndCallback: PropTypes.func,
+    getColumnGroupWidth: PropTypes.func,
+    columnKey: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number
+    ]),
+  };
+
+  /**
+   * Instance of DOMMouseMoveTracker to capture mouse events
+   * @type {DOMMouseMoveTracker}
+   */
+  mouseMoveTracker = null;
+
+  /**
+   * Displacement of reorder cell while reordering
+   * @type {number}
+   */
+  distance = 0;
+
+  /**
+   * Set true while reordering
+   * @type {boolean}
+   */
+  animating = false;
+
+  /**
+   * Frame Id of requested animation frame
+   * @type {number | null}
+   */
+  frameId = null;
+
+  /**
+   * Initial position of scroll
+   * @type {number}
+   */
+  scrollStart = 0;
+
+  /**
+   * Left of column
+   * @type {number}
+   */
+  originalLeft = 0;
+
+  render() {
+    const style = {
+      height: this.props.height,
+    };
+
+    return (
+      <div
+        className={cx({
+          'fixedDataTableCellLayout/columnReorderContainer': true,
+          'fixedDataTableCellLayout/columnReorderContainer/active': false,
+        })}
+        onMouseDown={this.onMouseDown}
+        onTouchStart={this.props.touchEnabled ? this.onMouseDown : null}
+        onTouchEnd={this.props.touchEnabled ? e => e.stopPropagation() : null}
+        onTouchMove={this.props.touchEnabled ? e => e.stopPropagation() : null}
+        style={style}>
+      </div>
+    );
+  }
+
+  /**
+   *
+   * @param {MouseEvent} event
+   */
+  onMouseDown = (event) => {
+    this.distance = 0;
+    this.animating = true;
+    this.scrollStart = getState().scrollX;
+    this.originalLeft = this.props.left;
+    this.initializeDOMMouseMoveTracker();
+    this.mouseMoveTracker.captureMouseMoves(event);
```

Comment:
This should be part of `this.initializeDOMMouseMoveTracker`

Reply:
I think `this.initializeDOMMouseMoveTracker()` should be moved to constructor because it creates a new instance everytime a ouseDown event happens

Reply:
Done